upgrade_test.py

```diff
@@ -203,7 +203,6 @@ def upgrade_node(self, node):
                              authorization_in_upgrade)
         check_reload_systemd_config(node)
         node.start_scylla_server()
```

Comment:
please use the parameter and put the number only for the upgrade test
and put a comment with the scylla issue link, there is a scylla issue for it starting to be that slow ?

Reply:
Currently we only use sct (master) to test upgrade Scylla 3.2 on Debian 9, and the timeout occurred many times.

And I didn't see the problem with sct branch-3.2 with Debian 9.

So we still need more testing to know if it's a general problem of only on Debian 9, if it's a scylla issue or sct master issue.

----

I just created a sct issue (https://github.com/scylladb/scylla-cluster-tests/issues/1726) and added a comment in the code.